packages/webapp/src/routes/land/upload-geospatial-file.js

```diff
@@ -70,6 +69,107 @@ const buildSignalRConfig = (sessionId, config) => {
   }
 }
 
+const performUpload = async (request, h) => {
+  const config = buildConfig(request.yar.id)
+
+  try {
+    const geospatialData = (await uploadFiles(logger, request, config))
```

Comment:
Why is the await uploadFiles() wrapped in parentheses? I think they can be removed, unless there is a programmatic reason I don't know of?

Reply:
Refactored